stock_package_info/models/product_packaging_template.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class ProductPackagingTemplate(models.Model):
+    _name = 'product.packaging.template'
+    _description = 'Product Packaging Template'
+    _rec_name = 'packaging_template_name'
+
+    packaging_template_name = fields.Char(required=True)
+    package_type = fields.Selection([
+        ('unit', 'Unit'),
+        ('pack', 'Pack'),
+        ('box', 'Box'),
+        ('pallet', 'Pallet'),
+    ],
+        required=True,
+    )
+    product_tmpl_id = fields.Many2one(
+        comodel_name='product.template',
+        string='Product',
+        index=True,
+        ondelete='cascade',
+    )
+    length = fields.Float(
+        digits=dp.get_precision('Stock Weight'),
+    )
+    length_uom_id = fields.Many2one(
```

Comment:
Create `_default_length_uom_id` and `default_weight_uom_id` methods to return defaults for all these required UOMs. In those methods return `product.product_uom_inch` and `product.product_uom_lb` respectively.